### DISCUSSION TITLE
Operation during network loss

### DISCUSSION THREAD

**Discussion by guydvir2** [arendst/Tasmota#11245] [Q&A] [answered]
Hi,

In a case 
A)of post power loss / outage, sometime not all infrastructures boot at same time, and even can be some minutes apart ( Wifi back on, MQTT and so on ), 
or 
B) in a case that for some reason WiFi / MQTT broker is lost for a significant amount of time 

During this periods, how does Tasmota works ? will it be stacked in "trying to restore network mode" ? It is a big issue, since my IOT device, which for now is not Tasmota based, operates during that periods to operate a garage door/ Electric window curtain / blinds regardless network.

So how does Tasmota operate in such cases ?

**Comment by ascillato2** [comment, marked as answer]
Tasmota will continue to work as normal, except if you have changed the WIFICONFIG setting. See this command in the docs for all the possible options.
Also, if you have set some timers, as Tasmota won't have a NTP server to sync the time, timers won't be triggered.
Also, if you have set SWITCHTOPIC, the behavior of the switches will change. See this command in the docs for all the possible options.
